### ISSUE TITLE
[Question] Are there plans to add the ‘Unity’ method of switching apps?

### ISSUE THREAD

**Issue by steve-todorov** [home-sweet-gnome/dash-to-panel#1741]
Developers of Ubuntu Dock have decided to add an option to show multiple windows of the same application as a virtual workspace in Gnome 43 (something that seems to have been a feature in Unity). Are there plans to add an option into dash-to-panel as well?

Original: 
![image](https://149366088.v2.pressablecdn.com/wp-content/uploads/2022/06/present.gif)

Improvement:
![image](https://i.imgur.com/zezHKdY.gif)

Sources:
https://www.omgubuntu.co.uk/2022/06/dash-to-dock-window-spread-feature-proposal
https://www.omgubuntu.co.uk/2022/08/ubuntu-22-10-release-new-features (search for `Ubuntu Dock Gains App Spread`)

**Comment by steve-todorov** [comment]
ping.

**Comment by steve-todorov** [comment]
pong.

**Comment by charlesg99** [comment]
An initial version of the window spread is now on the master branch! Only 2½ years later 😂

**Comment by steve-todorov** [comment]
Better late than never! The `master` branch works pretty well! Thanks! I think we can close? :D

**Comment by steve-todorov** [comment]
Let me see if I got it right -- the question is what should happen when the `workspace isolation` is disabled? 
I believe the Unity behavior was to show the windows of that type (i.e. terminal) across all workspaces.
So when you click on the icon it gives you the overview of all terminal windows across all workspaces. 
I'm a bit speaking from memory thought so I might be wrong.

**Comment by charlesg99** [comment]
Yes you got it right and I ended up going that way. Now if workspaces aren't isolated, all windows of the focused app will be presented on a single workspace and you won't be able to switch workspaces while in that overview mode. Hopefully that'll make it less confusing. Please give it a try, now I would say it is "feature complete" :)

**Comment by steve-todorov** [comment]
Just tried it -- works splendid! Awesome work!
